XenAdmin/Dialogs/ControlDomainMemoryDialog.cs

```diff
@@ -69,8 +69,8 @@ private void UpdateMaintenanceWarning()
             Host_metrics metrics = host.Connection.Resolve(host.metrics);

```

Comment:
same here, please clean up unused namespace in ControlDomainMemoryDialog.cs file